### PR TITLE
Add Default Install For Gitops & Pipeline Operators

### PR DESCRIPTION
**Pull request by Jdubrick**
This PR adds a Helm Chart that facilitates the deployment of the GitOps Operator & Pipelines Operator. This also creates an instance of ArgoCD in the default namespace.

Portions of this chart were referenced from RHTAP: https://github.com/redhat-appstudio/rhtap-installer/tree/main/chart

### CODE REVIEW

**Review by michael-valdron — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+## Installation
+
+Run `helm upgrade --install <release-name> <path-to-chart>` to deploy default installations of the necessary operators.
+
+#### Example Install
+
+`helm upgrade --install setup-default-operators ./chart`
+
+## Troubleshooting
```
Comment (thepetk) [inline README.md]:
```suggestion
### Uninstall Release
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
-# rhdh-installer
-Contains Helm Charts for installing GitOps/Pipeline Operators as well as installation scripts
+# AI RHDH Installer
+
+## Installation
+
+Run `helm upgrade --install <release-name> <path-to-chart>` to deploy default installations of the necessary operators.
+
+#### Example Install
```
Comment (thepetk) [inline README.md]:
```suggestion
### Example Install
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+
+`helm upgrade --install setup-default-operators ./chart`
+
+## Troubleshooting
+
+Since for the default installation the ServiceAccount is being deployed to the `default` namespace with admin permissions you are unable to remove it during a `helm uninstall`. You first need to remove the ServiceAccount manually before running `helm uninstall`
+
+Reference: https://access.redhat.com/solutions/7055600
+
+`oc delete sa helm-manager --as backplane-cluster-admin`
```
Comment (thepetk) [inline README.md:17]:
```suggestion
`oc delete sa helm-manager --as backplane-cluster-admin`

After the ServiceAccount is removed, run `helm uninstall <release-name>`
```

File: README.md (hunk 1 above)
Comment (thepetk) [reply README.md]: I'm only thinking that this is something like a removal guide with clear instructions, so I'd prefer `Uninstall release` or something similar which sounds less negative. But is very minor.

**Review by thepetk — COMMENTED** [review]
Left some very minor comments/thoughts. I'm also in the process of testing on a cluster of mine the provided helm charts :) :)

**Comment by thepetk** [comment]
Just tested it on a cluster of mine and it works fine. I was able to see the 2 installed operators.

**Review by thepetk — APPROVED** [review]
/lgtm

**Review by michael-valdron — APPROVED** [review]
/lgtm

**Pull request merged**